angel_system/impls/emotion_models/ComParE_model.py

```diff
@@ -0,0 +1,53 @@
+import importlib.util
```

Comment:
@maxlchen are these emotion files something we want in this PR? These were in the forked repo when I started and I suspect the answer is no.